tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/AbstractOSController.java

```diff
@@ -119,7 +126,7 @@ public boolean isSatisfied() {
       @Override
       public String describeFailure() {
         runSync(DUMP_CMD);
-        return format("The mule instance didn't start on time: %s", status);
+        throw new MuleControllerException(format("The mule instance didn't start on time: %s", status));
```

Comment:
Why would we do this?

Reply:
i needed to force an exception to catch in and print the log. Refactored this